runtime/redirector/forwarders.m4

```diff
@@ -411,6 +411,10 @@ _IF([JAVA_SPEC_VERSION >= 20],
 	[_X(JVM_VirtualThreadHideFrames, JNICALL, false, void, JNIEnv *env, jobject vthread, jboolean hide)])
 _IF([JAVA_SPEC_VERSION >= 21],
 	[_X(JVM_IsForeignLinkerSupported, JNICALL, false, jboolean, void)])
+_IF([JAVA_SPEC_VERSION >= 21],
+	[_X(JVM_VirtualThreadStart, JNICALL, false, void, JNIEnv *env, jobject vthread)])
+_IF([JAVA_SPEC_VERSION >= 21],
+	[_X(JVM_VirtualThreadEnd, JNICALL, false, void, JNIEnv *env, jobject vthread)])
 _IF([JAVA_SPEC_VERSION >= 21],
 	[_X(JVM_VirtualThreadMount, JNICALL, false, void, JNIEnv *env, jobject vthread, jboolean hide, jboolean firstMount)])
```

Comment:
The final parameter of `JVM_VirtualThreadMount()` and `JVM_VirtualThreadUnmount()` has been removed.

Reply:
I think this signature change is implemented by 
* https://github.com/eclipse-openj9/openj9/pull/17125

Reply:
Please remove those parameters; #17125 should not conflict with this no matter which order the two pull requests are merged.

Reply:
Removed `firstMount` in `JVM_VirtualThreadMount()`, `lastUnmount` in `JVM_VirtualThreadUnmount`, and passed `JNI_TRUE` to `JVM_VirtualThreadMount[Begin|End]` and `JVM_VirtualThreadUnmount[Begin|End]`, these are temporary implementation, expected to be replaced by https://github.com/eclipse-openj9/openj9/pull/17125.